queries/dictionary.py

```diff
@@ -155,9 +154,9 @@ def not_found() -> None:
         answ = ""
         # Generate list of the form "í fyrsta lagi a, í öðru lagi b, ..."
         for i, x in enumerate(df[: len(_ENUM_WORDS)]):
-            answ += f"{i + 1}. {x}\n"
+            answ += f"{i+1}. {x}\n"
             enum = f"í {_ENUM_WORDS[i]} lagi,"
-            voice += f"{enum} {x}, "
+            voice += f"{enum} {x.replace('<', '[').replace('>', ']')}, "
```

Comment:
Not entirely sure what's going on here.

Reply:
One example dictionary output:
```
Orðið „ár“ getur þýtt: í fyrsta lagi, áhald til að knýja bát með handafli, í
öðru lagi, tryggja hagsmuni sína, koma sér í góða aðstöðu, í þriðja lagi, gefast
upp, í fjórða lagi, aðstoða <hann>, í fimmta lagi, reyna með öllum ráðum <að fá
hann lausan>, í sjötta lagi, fullyrða (of) mikið, vera (of) stórorður,.
```
The speech synthesis doesn't allow stuff that looks like SSML tags and GreynirSSMLParser removes such stuff. This fix replaces <> with [], so the contained words are read out loud and/or don't cause errors during speech synthesis.

Reply:
I'm not quite sure why I decided to have brackets instead of just spaces though.